code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -209,79 +205,58 @@
 		if(!isnum(item) && query_tree[item])
 
 			if(istype(query_tree[item], /list))
-				usr << "[spaces]    ("
+				usr << "[spaces]&nbsp;&nbsp;&nbsp;&nbsp;("
 				SDQL_testout(query_tree[item], indent + 2)
-				usr << "[spaces]    )"
+				usr << "[spaces]&nbsp;&nbsp;&nbsp;&nbsp;)"
 
 			else
-				usr << "[spaces]    [query_tree[item]]"
-
-
+				usr << "[spaces]&nbsp;&nbsp;&nbsp;&nbsp;[query_tree[item]]"
```

Comment:
Actually shit I need to fix this because you don't have goonchat and BYOND's output controls almost certainly hate nbsp's.